bin/offline-dbg

```diff
@@ -133,6 +142,129 @@ EOF
     echo "**************************************************************"
 }
 
+check_for_OpenFlow_14() {
+    # If no protocol listed, default to atomic bundle txn (enabled by default on ovs 2.8 and on)
+    local hex_ofp_version=$(cat ${CMD_DIR}/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
+    local version=${open_flow_protocols[$hex_ofp_version]}
+    if [ -z "$version" ]; then
+        echo " --bundle"
+    else
+        [ ${version#OpenFlow} -ge 14 ] && bundle=" --bundle" || bundle=""
+        echo $bundle
+    fi                    
+}
+
+do_collect-sos() {
+    if [ $# -lt 1 ]; then
+        usage
+        exit 1
+    fi
+    local sos=$1
+
+    local dir_name=`tar -tf $sos | head -1 | cut -f1 -d"/"`
+
+    local sos_files="${dir_name}/sos_commands/rpm ${dir_name}/sos_commands/openvswitch"
+
+    local db_locations="var/lib/openvswitch etc/openvswitch usr/local/etc/openvswitch"
+
+    if test -f "$sos"; then
+        mkdir ${WORKDIR}/sos_report
+        echo "Extracting sos report from '$sos'..."
+    else
+        echo "SOS archive file not found. Please specify a <sosreport>.tar.xz archive"
+    fi
+
+    for dir in $sos_files; do
+        tar -xvf $sos -C ${WORKDIR}/sos_report $dir --strip-components 1 &>/dev/null || error "Could not extract critical directory $dir from $sos"
+    done
+    CMD_DIR=${WORKDIR}/sos_report/sos_commands/openvswitch/
+
+    local db_dir=''
+    for dir in $db_locations; do
+        tar -xvf $sos -C ${WORKDIR}/sos_report ${dir_name}/${dir}/conf.db --strip-components 1 &>/dev/null || true
+        if test -f "${WORKDIR}/sos_report/${dir}/conf.db"; then
+            db_dir=$dir
+            break
+        fi
+    done
+
+    if [ -z "${db_dir-}" ]; then
+        echo "warning: conf.db not found in default locations."
+        echo "If the target cluster has defined \$ovs_dbdir it could be located there"
+        echo "Please add manually using collect-db or add the .db directory to \$db_locations and re-run collect-sos"
+    else
+        cp ${WORKDIR}/sos_report/${db_dir}/conf.db ${WORKDIR}/db.conf
+    fi
+
+    # OVS 2.7 and earlier do not enable OpenFlow 1.4 (by default) and lack
+    # other features needed to save and restore flows.
+    local ovs_ver=$(cat ${WORKDIR}/sos_report/sos_commands/rpm/package-data | grep -o ^openvswitch-[0-9].[0-9]*.[0-9])
+    case $ovs_ver in
+    "openvswitch-1."*.*)
+        error "Outdated ovs version (2.8 or higher required)"
+        ;;
+    "openvswitch-2."[0-7].*)
+        error "Outdated ovs version (2.8 or higher required)"
+        ;;
+    esac
+
+    save_dir=${WORKDIR}/restore_flows
+    mkdir -p $save_dir
+    bridges=$(cat ${CMD_DIR}/ovs-vsctl_-t_5_list-br)
```

Comment:
Currently the way the script behaves if run against an old sos is:
```
./bin/offline-dbg -w /tmp/tmp1 collect-sos ~/devel/sosreports/ovn/sosreport-compute-0-2021-06-03-awkezkh.tar.xz 
Extracting sos report from '/home/amorenoz/devel/sosreports/ovn/sosreport-compute-0-2021-06-03-awkezkh.tar.xz'...
warning: conf.db not found in default locations.
If the target cluster has defined $ovs_dbdir it could be located there
Please add manually using collect-db or add the .db directory to $db_locations and re-run collect-sos
cat: /tmp/tmp1/sos_report/sos_commands/openvswitch//ovs-vsctl_-t_5_list-br: No such file or directory
```
We should check if these files exist to avoid the script just failing (because of the "set -e")